README.md

```diff
@@ -1,47 +1,43 @@
 Pega Docker Image
 ===========
 
-This project is a slim version of pega-web docker image that runs pega 8 platform.
-
-
-# Usability note
-
-Although this code supports running Pega8 in a tomcat based docker container, it does not mean that Pega8 can be used in a docker container clustering environment as is, such as Docker Swarm and Kubernetes.
+This project produces a base Docker image that can be used to create a complete Docker image for running Pega using containers.  The Pega .war file is not included in this build.
 
 # Using this image
 
-The image itself is not runnable directly because it does not come with the Pega 8
-web applications.  Therefore you must use this image as a base to construct an 
-executable Docker image.
+The image itself is not runnable directly because it does not come with the Pega web application (.war file).  Therefore you must use this image as a base to construct an executable Docker image.
 
 ## Constructing your image
 
-The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and 
-extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega 8 software libraries and specify a valid JDBC driver for your target database.
+The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database.
```

Comment:
It is recommended that this is done on a Linux system ==>It's a best practice to configure the deployment on a Linux system